docs/configuration/radio/device.mdx

```diff
@@ -17,25 +17,25 @@ Sets the role of the node.
+|      Value      | Descrption                              | Role Details                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| :-------------: | :-------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|    `CLIENT`     | Standard node                           | This role will follow the standard routing rules, while also allowing the device to interact with client applications via BLE/Wi-Fi (Android/Apple/Web/etc).                                                                                                                                                                                                                                                                                     |
+|  `CLIENT_MUTE`  | Dead-end node (sends own messages only) | Same as a client except packets will not hop over this node, does not contribute to routing packets for mesh.                                                                                                                                                                                                                                                                                                                                    |
+| `CLIENT_HIDDEN` | Hidden Client Node                      | Used for nodes that "only speak when spoken to." Turns off all of the routine broadcasts but allows for ad-hoc communication. Still rebroadcasts, but with local only rebroadcast mode (known meshes only). Can be used for clandestine operation or to dramatically reduce airtime and power consumption.                                                                                                                             |
+|    `ROUTER`     | Relay node                              | Acts like a powerful relay station, intended to be placed in high or open areas with better signal coverage. It helps pass messages across the network efficiently as other nodes will prioritize routing to this node first. This device won't interact with client apps and conserves energy by turning off certain features like Bluetooth, Wi-Fi, and the screen. _Note_: It might use more power due to its active role in message routing. |
```

Comment:
I believe instead of "other nodes will prioritize routing to this node first" it's more accurate that "this node will have priority when re-broadcasting"  If I understand correctly, there is less (random) delay before sending.

Reply:
Edited